### DISCUSSION TITLE
Camera History view (.14 RC1)

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#12801] [General]
While in home page "Live view" select a particular camera you get a live view of the selected camera, expected. When you select "History" tab along the top of this page, you get thumbnails for all the cameras along the bottom. I expected to see History for the selected camera.
Not sure if this is as intented. Seems to me the History tab does not belong on this page as it seem to be a repeat of the "Review" tab on the home page.

**Comment by hawkeye217** [comment]
Yes, this is intentional, and it's not a repeat of the Review tab.

The details are explained in the [release notes](https://github.com/blakeblackshear/frigate/releases/tag/v0.14.0-rc2).

> Review
> The Review area of Frigate 0.14 is for quickly reviewing historical footage of interest from your cameras. Review items are indicated on a vertical timeline and displayed as a grid of previews - bandwidth-optimized, low frame rate, low resolution videos. Hovering over or swiping a preview plays the video and marks it as reviewed. If more in-depth analysis is required, the preview can be clicked/tapped and the full frame rate, full resolution recording is displayed. Review items are filterable by date, object type, and camera.

> History
> In Frigate 0.14, full frame rate, full resolution camera footage (called "recordings" in earlier versions) can be viewed and scrubbed via a timeline. Previews of your other cameras are displayed and scrubbed alongside the main selected camera.

**Reply by 18923** [reply]
Thank you for the explanation. It all makes sense now.
